src/domain/app/routes/LocaleRoutes.tsx

```diff
@@ -36,47 +42,51 @@ const LocaleRoutes: React.FC<Props> = ({
 
   return (
     <PageLayout>
-      <Switch>
-        <Route
-          exact
-          path={getLocalePath(ROUTES.HOME)}
-          component={LandingPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.CREATE_EVENT)}
-          component={CreateEventPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.EVENT_SAVED)}
-          component={EventSavedPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.EDIT_EVENT)}
-          component={EditEventPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.EVENTS)}
-          component={EventsPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.LOGOUT)}
-          component={LogoutPage}
-        />
-        <Route
-          exact
-          path={getLocalePath(ROUTES.SEARCH)}
-          component={EventSearchPage}
-        />
-        <Route path={getLocalePath(ROUTES.HELP)}>
-          <HelpPageRoutes locale={locale} />
-        </Route>
-        <Route component={NotFound} />
-      </Switch>
+      <React.Suspense fallback={<div style={{ minHeight: '100vh' }} />}>
+        <Switch>
+          <Route
+            exact
+            path={getLocalePath(ROUTES.HOME)}
+            component={LandingPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.CREATE_EVENT)}
+            component={CreateEventPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.EVENT_SAVED)}
+            component={EventSavedPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.EDIT_EVENT)}
+            component={EditEventPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.EVENTS)}
+            component={EventsPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.LOGOUT)}
+            component={LogoutPage}
+          />
+          <Route
+            exact
+            path={getLocalePath(ROUTES.SEARCH)}
+            component={EventSearchPage}
+          />
+          <Route path={getLocalePath(ROUTES.HELP)}>
+            <HelpPageLayout>
+              <HelpPageRoutes locale={locale} />
+            </HelpPageLayout>
```

Comment:
Why was this (HelpPageLayout) moved here?

Reply:
The side navigation started to blink after Lazy loading HelpPageRoutes so I moved it here so the side navigation code is loaded on main JS chunk.